ios/RnCryptoTests/AesCipherTest.swift

```diff
@@ -6,77 +6,286 @@
+     let expectation = XCTestExpectation(description: "Large file encryption")
+     var encryptError: RnCryptoError?
+
+     sut.encryptToChunks(
+         input: inputStream,
+         outputs: outputStreams,
+         key: key,
+         iv: iv,
+         chunkSize: chunkSize
+     ) { error, status in
+         encryptError = error
+         expectation.fulfill()
+     }
+
+     wait(for: [expectation], timeout: 30.0)
+     
+     outputStreams.forEach { $0.close() }
+     inputStream.close()
+     
+     XCTAssertNil(encryptError)
+  }
+  
+  func testGigabyteEncryptToChunks() {
+     let gigabyte = 1024 * 1024 * 1024
+     let testData = Array(repeating: UInt8(1), count: gigabyte)
+     let key = Array(repeating: UInt8(0), count: 32)
+     let iv = Array(repeating: UInt8(0), count: 16)
+     let chunkSize = 35 * 1024 * 1024 // 35MB chunk
+     
+     let inputStream = InputStream(data: Data(testData))
+     let outputStreams = Array(repeating: OutputStream(toMemory: ()), count: 30)
+
+     inputStream.open()
+     outputStreams.forEach { $0.open() }
+     
+     let expectation = XCTestExpectation(description: "GB encryption")
+     var encryptError: RnCryptoError?
+
+     sut.encryptToChunks(
+         input: inputStream,
+         outputs: outputStreams,
+         key: key,
+         iv: iv,
+         chunkSize: chunkSize
+     ) { error, status in
+         encryptError = error
+         expectation.fulfill()
+     }
+
+     wait(for: [expectation], timeout: 60.0)
+     
+     outputStreams.forEach { $0.close() }
+     inputStream.close()
+     
+     XCTAssertNil(encryptError)
+  }
+  
+  func testEncryptDecryptCycle() {
+     // Setup: Create test data - 1MB
+     let megabyte = 1024 * 1024
+     let testData = Array(repeating: UInt8(1), count: 1 * megabyte)
+     let key = Array(repeating: UInt8(0), count: 32)
+     let iv = Array(repeating: UInt8(0), count: 16)
+     let chunkSize = 400 * 1024 // 400KB chunks
+
+     // Prepare input/output streams
+     let inputStream = InputStream(data: Data(testData))
+     let outputs = [
+         OutputStream(toMemory: ()),
+         OutputStream(toMemory: ()),
+         OutputStream(toMemory: ())
+     ]
+     
+     inputStream.open()
+     outputs.forEach { $0.open() }
+     
+     // 1. Test encryption in chunks
+     let expectation = XCTestExpectation(description: "Test")
+     var encryptError: RnCryptoError?
+
+     sut.encryptToChunks(input: inputStream, outputs: outputs, key: key, iv: iv, chunkSize: chunkSize) { error, status in
+         encryptError = error
+         expectation.fulfill()
+     }
+
+     wait(for: [expectation], timeout: 5.0)
+     XCTAssertNil(encryptError)
+     
+     // 2. Combine encrypted chunks
+     var encryptedData = Data()
+     for stream in outputs {
+         if let data = stream.property(forKey: .dataWrittenToMemoryStreamKey) as? Data {
+             encryptedData.append(data)
+         }
+     }
+     XCTAssertGreaterThan(encryptedData.count, 0)
```

Comment:
You should assert here the following: 

`encryptedData.count === testData.count`

As we are using AES, which is symmetric cryptography, this should match